docs/self-hosted/docker-guide.md

```diff
@@ -7,47 +7,35 @@ readTime: 3 min read
 
 ::: info Non-Docker Guides
 
-We only publish and maintain self hosting guides using Docker as this removes many environment-specific configuration
+We only publish and maintain self-hosting guides using Docker as this removes many environment-specific configuration
 problems. If you can't or don't want to use Docker, we also publish an
 [npm package](https://www.npmjs.com/package/directus) without guides.
 
 :::
 
-Directus is published to [Docker Hub](https://hub.docker.com/r/directus/directus) under `directus/directus`. To use the
-latest Directus image from Docker Hub, run:
+Directus is published to [Docker Hub](https://hub.docker.com/r/directus/directus) under `directus/directus`. If you're just getting started, check out our [Self-Hosting Quickstart](/self-hosted/quickstart.html).
 
-```bash
-# Make sure to change sensitive values (KEY, SECRET, ...) in production
-docker run \
-  -p 8055:8055 \
-  -e KEY=255d861b-5ea1-5996-9aa3-922530ec40b1 \
-  -e SECRET=6116487b-cda1-52c2-b5b5-c8022c45e263 \
-  directus/directus
-```
-
-### Installing Specific Versions
+## Installing Specific Versions
 
 To stick to a more specific version of Directus you can use one of the following tags:
 
 - Full version, e.g. `10.0.0`
 - Minor releases, e.g. `10.0`
 - Major releases, e.g. `10`
 
-To use a specific version of Directus, run:
+To use a specific version of Directus, include the version number in your `services.directus.image` value:
 
-```bash
-# Make sure to change sensitive values (KEY, SECRET, ...) in production
-docker run \
-  -p 8055:8055 \
-  -e KEY=255d861b-5ea1-5996-9aa3-922530ec40b1 \
-  -e SECRET=6116487b-cda1-52c2-b5b5-c8022c45e263 \
-  directus/directus:10.0.0
+```yml
+services:
+  directus:
+    image: directus/directus // [!code --]
+    image: directus/directus:10.0.0 // [!code ++]
 ```
 
-### Configure Admin User
+## Configure Admin User
 
-The published Docker image will automatically populate the database and create an admin user. To configure the
-email/password for this first user, pass the following env vars:
+The `ADMIN_EMAIL` and `ADMIN_PASSWORD` variables, while shown in the quickstart, are optional. If omitted, the published Docker image will automatically populate the database and create an admin user. To configure the
+email/password for this first user, include the following environment variables:
```

Comment:
We should probably add here that the automatically generated admin credentials will only be shown in the docker bootstrap logs when starting for the first time. And perhaps mention that you can just add the ADMIN_* settings for the first startup and remove them from the configuration after.

Reply:
But can't you then go and change them once the db has been seeded for the first time? Doesn't that make more sense? To explicitly say GO CHANGE THEM.

Reply:
If you've missed the password in logs then you can only change that password by creating a new admin user using the CLI. because you dont have access to the `admin@example.com` mailbox to reset the password and cant login to change it.

Reply:
Gotcha. Will add. Feels like the kind of tiny thing that could cause someone grief.